### DISCUSSION TITLE
WSL Installation Failure

### DISCUSSION THREAD

**Discussion by JusticeSol** [Cyfrin/foundry-full-course-cu#2368] [General]
Hello guys, I'm stocked at this point because i can't access my BIOS password that will enable me activate VIRTUALIZATION
If anyone knows an alternative route i'll be happy to get a guideline. 

Also,  i'll like to know if anyone has used the Git Bash method for Foundry Setup?

<img width="767" alt="wsl error message" src="https://github.com/user-attachments/assets/3d544f5b-f641-4b37-b78f-aa1d7fac3d65">

**Comment by EngrPips** [comment]
Hello @cromewar, any help here will be appreciated.

**Reply by JusticeSol** [reply]
Thanks @EngrPips 
Have you Installed yours?

**Reply by EngrPips** [reply]
You are welcome, Friend. I am working on Mac so i have no idea how to instal wsl.

**Reply by JusticeSol** [reply]
Oh that's fine, big ups fam.

**Reply by EngrPips** [reply]
Anytime mah man
